test/up-cmd.test.js

```diff
@@ -157,6 +157,49 @@ describe('Integration test for up command with helix pages', function suite() {
     assert.strictEqual(opened, `http://localhost:${port}/`);
   });
 
+  it('up command correctly replaces variables in url', async () => {
+    const MockedCommand = await esmock('../src/up.cmd.js', {
+      '../src/abstract-server.cmd.js': await esmock('../src/abstract-server.cmd.js'),
+    });
+    initGit(testDir, 'https://github.com/adobe/dummy-foo.git', 'test');
+    const cmd = new MockedCommand()
+      .withLiveReload(false)
+      .withDirectory(testDir)
+      .withOpen(false)
+      .withHttpPort(0)
+      .withUrl('https://main--{{repo}}--{{owner}}.hlx.page');
```

Comment:
I would try something completely different. because the example above is also the default url.

eg:
```suggestion
      .withUrl('https://{{repo}}--{{owner}}.custom.com');
```